Lob-API-public.yml

```diff
@@ -248,184 +250,699 @@ tags:
+          <td style="white-space: nowrap">A postcard is successfully created (Lob returns a 200 status code).</td>
+        </tr>
+        <tr>
+          <td style="white-space: nowrap"><code>postcard.rendered_pdf</code></td>
+          <td style="white-space: nowrap"><code>false</code></td>
+          <td style="white-space: nowrap">A postcard's PDF proof is successfully rendered.</td>
+        </tr>
+        <tr>
+          <td style="white-space: nowrap"><code>postcard.rendered_thumbnails</code></td>
+          <td style="white-space: nowrap"><code>false</code></td>
+          <td style="white-space: nowrap">A postcard's thumbnails are successfully rendered.</td>
+        </tr>
+        <tr>
+          <td style="white-space: nowrap"><code>postcard.deleted</code></td>
+          <td style="white-space: nowrap"><code>false</code></td>
+          <td style="white-space: nowrap">A postcard is successfully canceled.</td>
+        </tr>
+        <tr>
+          <td style="white-space: nowrap"><code>postcard.mailed</code></td>
+          <td style="white-space: nowrap"><code>true</code></td>
+          <td style="white-space: nowrap">A postcard receives a "Mailed" [tracking event](#operation/tracking_event). Only enabled for certain [Print & Mail Editions](https://dashboard.lob.com/#/settings/editions).</td>
+        </tr>
+        <tr>
+          <td style="white-space: nowrap"><code>postcard.in_transit</code></td>
+          <td style="white-space: nowrap"><code>true</code></td>
+          <td style="white-space: nowrap">A postcard receives an "In Transit" [tracking event](#operation/tracking_event).</td>
+        </tr>
+        <tr>
+          <td style="white-space: nowrap"><code>postcard.in_local_area</code></td>
+          <td style="white-space: nowrap"><code>true</code></td>
+          <td style="white-space: nowrap">A postcard receives an "In Local Area" [tracking event](#operation/tracking_event).</td>
+        </tr>
+        <tr>
+          <td style="white-space: nowrap"><code>postcard.processed_for_delivery</code></td>
+          <td style="white-space: nowrap"><code>true</code></td>
+          <td style="white-space: nowrap">A postcard receives a "Processed for Delivery" [tracking event](#operation/tracking_event).</td>
+        </tr>
+        <tr>
+          <td style="white-space: nowrap"><code>postcard.re-routed</code></td>
+          <td style="white-space: nowrap"><code>true</code></td>
+          <td style="white-space: nowrap">A postcard receives a "Re-Routed" [tracking event](#operation/tracking_event).</td>
+        </tr>
+        <tr>
+          <td style="white-space: nowrap"><code>postcard.returned_to_sender</code></td>
+          <td style="white-space: nowrap"><code>true</code></td>
+          <td style="white-space: nowrap">A postcard receives a "Returned to Sender" [tracking event](#operation/tracking_event).</td>
+        </tr>
+      </table>
+
+      <h3>Self Mailers</h3>
```

Comment:
It's prob easier for you to just ctrl-f and replace the rest of the tracking event links in the rest of the Event Type tables